appveyor.yml

```diff
@@ -11,11 +11,11 @@ environment:
 
     matrix:
         - PYTHON: C:\Python27
-          TOX_ENV: py27-test
+          TOX_ENV: py27-testwin32
         - PYTHON: C:\Python34
-          TOX_ENV: py34-test
+          TOX_ENV: py34-testwin32
         - PYTHON: C:\Python35
-          TOX_ENV: py35-test
```

Comment:
We came to the same conclusions 😃 
In fact I would just need a pure python lib that would implement [C14N](https://www.decalage.info/en/python/lxml-c14n), but `lxml` is the only lib to do this for now.